readme.md

```diff
@@ -132,13 +132,15 @@ Any of the [`https.request`](https://nodejs.org/api/https.html#https_https_reque
 
 Type: `string | URL`
 
-When specified, `prefixUrl` will be prepended to `url`. The prefix can be any valid URL, either relative or absolute. A trailing slash `/` is optional, one will be added automatically, if needed, when joining `prefixUrl` and `url`. The `url` argument cannot start with a `/` when using this option.
-
-Useful when used with `got.extend()` to create niche-specific Got-instances.
+When specified, `prefixUrl` will be prepended to `url`. The prefix can be any valid URL, either relative or absolute. A trailing slash `/` is  - one will be added automatically.
 
 **Note:** `prefixUrl` will be ignored if the `url` argument is a URL instance.
 
-**Tip:** If the input URL still contains the initial `prefixUrl`, you can change it as many times as you want. Otherwise it will throw an error.
+**Note:** Leading slashes in `input` are disallowed when using this option to enforce consistency and avoid confusion about how the `input` URL is handled, given that `input` will not follow the normal URL resolution rules when `prefixUrl` is being used, which changes the meaning of a leading slash.
+
+**Tip:** Useful when used with `got.extend()` to create niche-specific Got-instances.
+
+**Tip:** If the input URL still contains the initial `prefixUrl`, you can change it as many times as you want inside hooks. Otherwise it will throw an error.
```

Comment:
> If the input URL still contains the initial `prefixUrl`

Not clear to me what this exactly means.

Reply:
```suggestion
**Tip:** If the `prefixUrl` is included in the URL, you can change it as many times as you want inside hooks. Otherwise it will throw an error.
```

Reply:
```suggestion
**Tip:** You can change `prefixUrl` using hooks as long as the URL includes it. If the URL doesn't include it anymore, it will throw.
```